libs/providers/flagd-web/README.md

```diff
@@ -0,0 +1,55 @@
+# Client-side flagD Provider for OpenFeature
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+
+Flagd is a simple daemon for evaluating feature flags.
+It is designed to conform to OpenFeature schema for flag definitions.
+This repository and package provides the client code for interacting with it via the OpenFeature JavaScript SDK.
+
+## Installation
+
+```
+$ npm install @openfeature/flagd-web-provider
+```
+
+## Usage
+
+The `FlagdWebProvider` communicates with flagd via the connect protocol.
+Options can be defined in the constructor or as environment variables, with constructor options having the highest precedence.
+
+### Available options
+
+| Option name | Environment variable name | Type    | Default   |  Description   | 
```

Comment:
The environment variable section no longer makes sense.

Reply:
https://github.com/open-feature/js-sdk-contrib/pull/142/files/970bef1ffc114c093b91cacc6581cb9d80f3e859#r1013544350

Reply:
Fixed.